redis/src/cluster_async/mod.rs

```diff
@@ -699,20 +711,30 @@ where
 
     async fn try_cmd_request(
```

Comment:
This function is a bit hard to understand. can you add some comments, e.g., in the first part - single node request without routing, second - multi node request, third - single node request with routing. something that will explain it a bit more, I got lost between the CommandRouting, routing, route_option, RoutingInfo...

Reply:
added some comments, please tell me if they are sufficient.

Reply:
looks great now, thanks